lib/util/AuthConfig.js

```diff
@@ -94,7 +95,15 @@ class AuthConfig extends Config {
 
     constructor(location, path = null) {
         const configPath = (location == AuthConfig.LOCATION.GLOBAL) ? Osenv.home() : path;
-        super(Config.TYPE.AUTH, AUTH_CONFIG_FILE_NAME, configPath);
+
```

Comment:
Is there a reason we aren't using the super constructor the way that it was being used before?

We can pass in ternary statements, and override anything we want to do after the `super()` has done its thing.

Reply:
Yes, the super constructor calls some other functions that will fail using our current format of the `.impt.auth` file

Reply:
Other functions in AuthConfig?  Which ones?

Reply:
No, functions in Config, specifically _parse (which depends on the old format of `.impt.auth`, which I've overridden in AuthConfig so that when we initialize a new AuthConfig object,  the process won't fail.